config/groovy/util.groovy

```diff
@@ -54,7 +54,35 @@ switch(cleanerArgs[0]) {
         } else {
             // Note: processCustomRemote also drops one of the array elements from cleanerArgs
             cleanerArgs = common.processCustomRemote(cleanerArgs)
-            common.retrieve cleanerArgs, recurse
+            ArrayList<String> selectedModules = new ArrayList<String>()
+            String[] moduleList = common.retrieveAvailableItems()
+
+            for (String arg : cleanerArgs) {
+                if (!arg.contains('*') && !arg.contains('?')) {
+                    selectedModules.add(arg)
+                } else {
+                    for (String module : moduleList) {
+                        /**
+                         * Forming the regex:
```

Comment:
Could this please be moved to a method in `common.groovy` ? Then you can attach the javadoc properly and we can keep `util.groovy` more slim. Somewhat of a code spell to start bloating logic inside a switch, cleaner to just deal solely with the logic about where to delegate.